app/console.go

```diff
@@ -61,33 +65,80 @@ func RunConsole(a *App) *ishell.Shell {
 			toggleMiner(ctx, a)
 		},
 	})
+	shell.AddCmd(&ishell.Cmd{
+		Name: "user",
+		Help: "view or edit current user's info",
+		Func: func(ctx *ishell.Context) {
+			editUser(ctx, a)
+		},
+	})
 
+	// Set interrupt handler
+	shell.Interrupt(func(ctx *ishell.Context, count int, input string) {
+		ctx.Println("Saving app state and flushing logs...")
+		a.onExit()
+	})
 	shell.Start()
 	emoji.Println(":cloud: Welcome to the :sunny: Cumulus console :cloud:")
 	return shell
 }
 
-func create(ctx *ishell.Context, app *App) {
-	choice := ctx.MultiChoice([]string{
-		"Wallet",
-		"Transaction",
-	}, "What would you like to create?")
-	if choice == 0 {
-		createWallet(ctx, app)
+func send(ctx *ishell.Context, app *App) {
+	if len(ctx.Args) < 2 {
+		ctx.Println("Usage: send [amount] [public address]")
+		return
+	}
+
+	amount, err := strconv.ParseFloat(ctx.Args[0], 64)
+	if err != nil {
+		ctx.Println(err)
+		return
+	} else if amount <= 0 {
+		ctx.Println("Amount must be a positive decimal value")
+		return
+	}
+	amount *= float64(blockchain.CoinValue)
```

Comment:
I think this conversion is more complicated and probably requires a helper function to deal with edge cases